footer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+include('./header.php')
+?>
```

Comment:
Lets keep these file separated for future use. Do not include header in this

Reply:
resolved..pls check it